src/transformers/models/biogpt/modeling_biogpt.py

```diff
@@ -36,8 +45,14 @@
 _CHECKPOINT_FOR_DOC = "microsoft/biogpt"
 _CONFIG_FOR_DOC = "BioGptConfig"
 
+# SequenceClassification docstring
+_CHECKPOINT_FOR_SEQUENCE_CLASSIFICATION = _CHECKPOINT_FOR_DOC
+_SEQ_CLASS_EXPECTED_LOSS = 1.71
+_SEQ_CLASS_EXPECTED_OUTPUT = "'LABEL_0'"
```

Comment:
```suggestion
```
Let's not put the loss and output for a random model, and no need to have two variables with the same thing.